Mathlib/Analysis/NormedSpace/Banach.lean

```diff
@@ -95,8 +95,7 @@ theorem exists_approx_preimage_norm_le (surj : Surjective f) :
   rcases this with ⟨n, a, ε, ⟨εpos, H⟩⟩
   rcases NormedField.exists_one_lt_norm 𝕜 with ⟨c, hc⟩
   refine' ⟨(ε / 2)⁻¹ * ‖c‖ * 2 * n, _, fun y => _⟩
-  · refine' mul_nonneg (mul_nonneg (mul_nonneg _ (norm_nonneg _)) (by norm_num)) _
-    exacts [inv_nonneg.2 (div_nonneg (le_of_lt εpos) (by norm_num)), n.cast_nonneg]
+  · positivity
```

Comment:
could you move this one to the `refine'` (and change it to `refine`), to avoid one indentation layer?